### PR TITLE
259-tco

### PR DESCRIPTION
**Pull request by MRomalde**
TCO updated to the project, added GDPR Officer salary and the explication of Incomes.

### CODE REVIEW

File: PPL/costAnalisis.md [hunk 1]
```diff
@@ -0,0 +1,1724 @@
...
+#
+
+<br/>
+
+# 1. Costes del periodo de desarrollo
+
+
+## 1.1. Coste salarial del equipo (desarrollo)
+
+En este apartado se desglosan los costes salariales del equipo de desarrollo que contempla desde la primera semana de vida del proyecto hasta su puesta en producción, en este caso, 15 semanas. Hay que añadir que los costes de Community Manager serían computados a un trabajador externo ya que ningún miembro del equipo trabaja bien ese conjunto. Es por esto último que en la segunda tabla no se tiene en cuenta los costes salariales del Community Manager al ser un trabajador externo.
```
Comment (FelixoGudiel) [inline PPL/costAnalisis.md:143]: What do you mean by this? i dont understand the community manager section

File: PPL/costAnalisis.md [hunk 2]
```diff
@@ -0,0 +1,1724 @@
...
+   </td>
+   <td colspan="2" >6.619,20€
+   </td>
+  </tr>
+</table>
+
+<br/>
+
+
+<table>
```
Comment (FelixoGudiel) [inline PPL/costAnalisis.md:354]: Though I agree that community manager is outside of the team expenses, please update this table with the salary of the CM

File: PPL/costAnalisis.md [hunk 3]
```diff
@@ -0,0 +1,1724 @@
...
+<br/>
+
+## 1.7. Desglose total de los costes de desarrollo.
+
+En este subapartado están resumidos todos los costes asociados al periodo de desarrollo del producto.
+
+
+<table>
+  <tr>
+   <td colspan="2" ><strong>Costes totales del desarrollo (sin reservas de contingencia)</strong>
```
Comment (FelixoGudiel) [inline PPL/costAnalisis.md:1029]: this table should be updated with the new total value of the salaries as well

File: PPL/costAnalisis.md [hunk 4]
```diff
@@ -0,0 +1,1724 @@
...
+   </td>
+   <td><strong>54.920,48€</strong>
+   </td>
+  </tr>
+</table>
+
+<br/>
+
+
+<table>
```
Comment (FelixoGudiel) [inline PPL/costAnalisis.md:1073]: this table as well

File: PPL/costAnalisis.md [hunk 5]
```diff
@@ -0,0 +1,1724 @@
...
+Teniendo esto en cuenta, durante 37 semanas, que junto con las 15 semanas de desarrollo completamos el 1.er año de vida del producto, amortizamos **7.400€.**
+
+<br/>
+
+## 2.7 Desglose total de los costes de producción
+
+Como aclaración, el coste de periodo de mantenimiento es el salario **BRUTO**, no neto.
+
+
+<table>
```
Comment (FelixoGudiel) [inline PPL/costAnalisis.md:1506]: GDPR costs should be added to this table

File: PPL/costAnalisis.md [hunk 6]
```diff
@@ -0,0 +1,1724 @@
...
+Para el correcto análisis del TCO tendremos en cuenta los costes asociados al desarrollo y los costes de la puesta en producción de la aplicación durante  los dos primeros años de vida (15 semanas de desarrollo + **89 semanas**).
+
+Para ello calculamos los costes salariales del equipo encargado del mantenimiento y actualización de la aplicación una vez puesta en producción, se contará con un equipo de 5 desarrolladores junior y un senior con una jornada laboral de 10 horas semanales.
+
+También tendremos en cuenta las compensaciones por trabajo en remoto de los trabajadores y los costes de las licencias requeridas.
+
+Por último, se ha analizado y tenido en cuenta para el TCO, los costes de marketing y publicidad, utilizando el mismo personal externo contratado durante el periodo de desarrollo, con las mismas condiciones laborales, es decir, con un contrato de 10h semanales, y con la misma retribución.
+
+
+<table>
```
Comment (FelixoGudiel) [inline PPL/costAnalisis.md:1610]: with the previous values, update this table as well

File: PPL/costAnalisis.md [hunk 7]
```diff
@@ -0,0 +1,1724 @@
...
+  </tr>
+</table>
+
+
+Si además queremos ver el Capex y el Opex, vamos a calcular lo que equivale a cada uno. Lo primero que vamos a ver es:
+
+- **Capex:** Representa el gasto inicial necesario para adquirir, construir o mejorar los activos físicos de un proyecto. Incluye la compra de equipos, maquinaria, infraestructura, y otros activos que se capitalizan y se amortizan a lo largo del tiempo. Los gastos de CAPEX son típicamente de naturaleza no recurrente y se consideran como inversiones a largo plazo en el negocio.
+- **Opex:** Representa los costos recurrentes y continuos asociados con el funcionamiento y mantenimiento de los activos adquiridos con el CAPEX. Incluye gastos como salarios del personal, mantenimiento, suministros, alquileres, servicios públicos, y otros costos operativos necesarios para mantener en funcionamiento el proyecto. Los gastos de OPEX son considerados como gastos corrientes y se incurren durante toda la vida útil del proyecto.
+
+Para el Capex:
```
Comment (FelixoGudiel) [inline PPL/costAnalisis.md:1653]: Finally, update the "Sueldos menusales en desarrollo" in Capex and add a line for GDPR in Opex

File: PPL/costAnalisis.md [hunk 8]
```diff
@@ -0,0 +1,1724 @@
...
+| :-----------:|:--------------: | :----------- |
+|Optimista   |   43.545,6€            |1.045.094,4€   |
+|Realista   |21.772,8€            |522.547,2€   |
+|Pesimista|7.257,6€           |174.182,4€   |
+
+
+Usando las estimaciones obtenidas en el documento "Estimación de Ingresos" podemos crear un gráfico que compara los costes que se acaban de mostrar con los posibles ingresos a recibir.
+
+
+<image src="./images/grafico-tco.png" alt="Grafico de los costes asociados">
```
Comment (FelixoGudiel) [inline PPL/costAnalisis.md:1717]: this image does not show

**Review by FelixoGudiel — CHANGES_REQUESTED** [review]
Really good job with the estimates! They seem reasonable. However, this added costs have not been added to the tables so some of them are incorrect. Please add these costs, it should not be a very hard task as I have marked the incorrect tables. Also, one image is not working.

File: PPL/costAnalisis.md (hunk 1 above)
Comment (MRomalde) [reply PPL/costAnalisis.md:143]: This was the last version, did not updated it, my bad it will be updated tomorrow!

**Review by fracalrod3 — CHANGES_REQUESTED** [review]
As Félix states, most tables need to be thoroughly checked and updated, but I must say that on top of everything he said, It'd be much appreciated if you broke down what's been added in each version from 1.0.5 (included) onwards.

File: PPL/costAnalisis.md (hunk 2 above)
Comment (MRomalde) [reply PPL/costAnalisis.md:354]: done it!

File: PPL/costAnalisis.md (hunk 4 above)
Comment (MRomalde) [reply PPL/costAnalisis.md:1073]: is already updated too!

File: PPL/costAnalisis.md (hunk 6 above)
Comment (MRomalde) [reply PPL/costAnalisis.md:1610]: same as before, is included in the personal and its updated!

File: PPL/costAnalisis.md (hunk 7 above)
Comment (MRomalde) [reply PPL/costAnalisis.md:1653]: is updated already because Community manager has the same salary as junior!

File: PPL/costAnalisis.md (hunk 8 above)
Comment (MRomalde) [reply PPL/costAnalisis.md:1717]: already updated!

**Comment by MRomalde** [comment]
thanks for the feedback! I updated all the things that you said. Please check if something is wrong !

**Comment by MRomalde** [comment]
I want to add one thing more, we changed and we are now 11 Junior Developper, with 2 Community manager and with the GDPR officer now! the salary of the GDPR is added to the salarys in production

**Review by FelixoGudiel — COMMENTED** [review]
You have fixed everything I said so fast! I am very happy with the end result, but as I was re-reading the document, I have found that we will only pay for community managers during the develop phase. I find this a bit confusing. Are we expecting that the NGO will provide the community manager? I want to know your take on this!

**Comment by MRomalde** [comment]
thanks for the feedback! the costs was already updated because one of our junior developer will be the community manager, so, because the salary is the same, the costs are already updated, we just needed to fix the table of salary in the file

**Review by FelixoGudiel — APPROVED** [review]
Nice job!

**Review by fracalrod3 — APPROVED** [review]
Sorry for taking so long to re-review It, I see you've fixed everything important

**Pull request merged**